opencdms_process/process/climatol/__init__.py

```diff
@@ -19,29 +19,131 @@ def windrose(obs):
         os.path.dirname(__file__),
         'windrose.r',
     )
-    r.source(script)
-
     with localconverter(ro.default_converter + pandas2ri.converter):
         _obs = ro.conversion.py2rpy(obs)
 
     # _windrose = ro.globalenv['windrose']
     # r(_windrose(_obs, 'station code', 'station name'))
     ro.globalenv['observations'] = _obs
 
-    r('''
+    r(f"""
```

Comment:
Ideally the R-script would not be pasted modified into the Python script and would instead be sourced (because many of the R processes will be maintained externally by other projects).

Was there a reason for doing this?  Are you using string substitution some where?

Reply:
I was getting this error `Parsing status not OK - PARSING_STATUS.PARSE_ERROR` when sourcing from file. I will try to figure out why that error occurred and how to solve it.

Reply:
Thank you - the error doesn't sound familiar to me. The only thing I can think of could be a problem with (Windows) line endings, but it's probably not that.